src/components/toast/toast.stories.mdx

```diff
@@ -440,6 +440,60 @@ Toast variant is `success` by default
   </Story>
 </Canvas>
 
+### Dismissible without autoFocus
+
+<Canvas>
+  <Story name="dismissible without autoFocus" parameters={{ info: { disable: true } }}>
+    {() => {
+      const onDismissClick = (evt) => {
+        if (!isOpen) {
+          window.scrollTo(0, 0);
+        }
+        setIsOpen(!isOpen);
+        action("click")(evt);
+      };
+      const [isOpen, setIsOpen] = useState(false);
```

Comment:
I think this would be a good story to let chromatic capture so we can confirm the close icon isn't autoFocused, I'd update this to be

```
import isChromatic from "relative path/isChromatic";
export const isOpenForChromatic = isChromatic();
...
      const [isOpen, setIsOpen] = useState(isOpenForChromatic);
```